get.js

```diff
@@ -160,12 +152,10 @@ function getStream (cache, key, opts) {
       const src = read.readStream(
         cache,
         entry.integrity,
-        opts.concat({
-          size: opts.size == null ? entry.size : opts.size
-        })
+        { ...opts, size: size == null ? entry.size : size }
```

Comment:
Extremely minor nit: `typeof size !== number ? entry.size : size` would be better here than the `==` trick or null and undefined.  (Leaving it as-is is also fine, though.)